frontend/titlebar.js

```diff
@@ -18,7 +18,6 @@ const init_titlebar = function() {
 
     const nav_main = document.querySelectorAll(".nav-main");
     nav_main.forEach((item) => {
-        console.log("click");
```

Comment:
Is this needed for functionality or is it not ment to be removed at a later date?